packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -27,6 +27,9 @@ const Color _kLightThemeSplashColor = const Color(0x66C8C8C8);
 const Color _kDarkThemeHighlightColor = const Color(0x40CCCCCC);
 const Color _kDarkThemeSplashColor = const Color(0x40CCCCCC);
 
+/// Holds the current attribute values for a particular theme.
```

Comment:
I'd be more specific about what sort of attributes.  For example, "Color and typography values for a material design theme."